docs/getting-started.md

```diff
@@ -31,7 +31,7 @@
 - An AWS IAM role to give to the Cluster API control plane.
 - [Minikube][minikube]
```

Comment:
We should probably specify a minimum version, since we require k8s v1.12.x

Reply:
Done. Thanks.